src/input/input.directive.ts

```diff
@@ -19,9 +19,14 @@ export class TextInput {
 	 * `light` or `dark` input theme
 	 */
 	@Input() theme: "light" | "dark" = "dark";
+	@Input() disabled = false;
```

Comment:
Do we need to add this? I _think_ Angular already lets us bind to the native `disabled` attribute